ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -52,12 +57,70 @@
 /*
  * PKCS11_CMD_PING		Acknowledge TA presence and return version info
  *
- * Optinal invocation parameter (if none, command simply returns with success)
+ * [in]         memref[0] = 32bit, unused, must be 0
+ * [out]        memref[0] = 32bit return code, enum pkcs11_rc
  * [out]        memref[2] = [
  *                      32bit version major value,
  *                      32bit version minor value
  *                      32bit version patch value
  *              ]
  */
 #define PKCS11_CMD_PING				0
+
+/*
+ * Command return codes
+ * PKCS11_CKR_<x> relates cryptoki CKR_<x> in meaning if not in value.
+ */
+enum pkcs11_rc {
+	PKCS11_CKR_OK				= 0,
+	PKCS11_CKR_GENERAL_ERROR		= 0x0001,
```

Comment:
I will change this using the pkcs11 pristine IDs as per https://github.com/OP-TEE/optee_client/pull/188#discussion_r378226433.